### ISSUE TITLE
I have some issue with runing app.py

### ISSUE THREAD

**Issue by EURAISA** [adamz799/Paints#1]
![image](https://user-images.githubusercontent.com/70467859/91655052-f1e74800-eae8-11ea-99cd-70c6726581fe.png)
I don't know where should I put the data on, create a new directory?
![image](https://user-images.githubusercontent.com/70467859/91655086-2d821200-eae9-11ea-993b-862a112650db.png)
what's this mean of green code, what's the format of data, should I creat 3 directory of those green name?
could I use chinese to share my problem

**Comment by adamz799** [comment]
> ![image](https://user-images.githubusercontent.com/70467859/91655052-f1e74800-eae8-11ea-99cd-70c6726581fe.png)
> I don't know where should I put the data on, create a new directory?
> ![image](https://user-images.githubusercontent.com/70467859/91655086-2d821200-eae9-11ea-993b-862a112650db.png)
> what's this mean of green code, what's the format of data, should I creat 3 directory of those green name?
> could I use chinese to share my problem

Yes, you need to create 3 folders as the code shows if you want to train the neuralnet, 'bw/' and 'color/' are the directories for input data(black&white and corresponding color images). But i don\`t remeber what 'block/' means,  because I haven\`t worked on Deep Learning for over a year. Anyway, all these directories are only used in traning process. To run the demo, you need to prepare a black&white lineart and a reference image to fetch color and follow the instruction:
![image](https://user-images.githubusercontent.com/20186028/91656189-5f937400-eae9-11ea-8d9f-7df3fb3100b4.png)

**Comment by adamz799** [comment]
> ![image](https://user-images.githubusercontent.com/70467859/91655052-f1e74800-eae8-11ea-99cd-70c6726581fe.png)
> I don't know where should I put the data on, create a new directory?
> ![image](https://user-images.githubusercontent.com/70467859/91655086-2d821200-eae9-11ea-993b-862a112650db.png)
> what's this mean of green code, what's the format of data, should I creat 3 directory of those green name?
> could I use chinese to share my problem

And, of course you can use Chinese.

**Comment by EURAISA** [comment]
感谢您的解答，我想block应该是指作为参考的色块，app.py是demo的话utils_app.py就是用来训练的吗？

**Comment by EURAISA** [comment]
如果可以的话我希望你能告诉哦我如何训练这个网络，我看了一遍代码只看见了使用参考图片和线稿路径的代码

**Comment by EURAISA** [comment]
I have try to run the demo not train the neural but it still warning me of  this , why and I have creat 3 folders
![image](https://user-images.githubusercontent.com/70467859/93095461-17c23e80-f6de-11ea-8e47-43ecdc7fa434.png)

**Comment by adamz799** [comment]
> 如果可以的话我希望你能告诉哦我如何训练这个网络，我看了一遍代码只看见了使用参考图片和线稿路径的代码

不好意思，最近工作比较忙，没来得及弄这个，训练的python文件我也不确定是哪个，因为当时用了很多种训练方法，可能需要你自己看一下。我记得当时比较有效的办法是把色域从RGB转换到HSV，然后用GAN分通道训练。文件路径的话，就是按照readme建立路径，如果找不到路径，有可能是启动位置不对，可以试试直接在cmd启动。代码我上传到train文件夹了，可以参考一下，有问题欢迎继续留言。

**Comment by EURAISA** [comment]
感谢上传，我是一名研究生，刚进入这个领域有很多还不懂，谢谢你的回复
